### PR TITLE
Adding the build portion for the provider and schema

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,16 +3,20 @@ PROJECT_NAME := Pulumi Amazon Web Services (AWS) EKS Components
 VERSION         := $(shell pulumictl get version)
 TESTPARALLELISM := 12
 
+PACK			:= eks
```
Comment (justinvp) [inline Makefile:6]: Nit: odd spacing

**Review by justinvp — APPROVED** [review]
Looks great!

File: Makefile (hunk 1 above)
Comment (stack72) [reply Makefile:6]: ah I see it's tabs not spaces 🙄 - I will fix that up in the next set of work required here :)

**Review by justinvp — APPROVED** [review]
Latest commits LGTM

**Pull request merged**